docs/introduction.rst

```diff
@@ -4,8 +4,11 @@ Introduction
 
 We illustrate with an example how the OGGM workflow is applied to the
 Tasman Glacier in New Zealand (see figure below).
-Here we describe shortly the purpose of each processing step, while more
-details are provided in the other sections:
+Here we describe shortly the purpose of each processing steps, while more
+details are provided in the other sections and the theory of glacier physic
+can for example be found in `Cuffey and Paterson, (2010)`_.:
+
+.. _Cuffey and Paterson, (2010): https://www.elsevier.com/books/the-physics-of-glaciers/cuffey/978-0-12-369461-4
```

Comment:
Maybe not scare people with Cuffey and Paterson right in the general intro? I like the reference to the other sections though.

Reply:
I understand not to scare people. However it makes more strength to the model with this reference ?